includes/Settings/SettingsDebug.php

```diff
@@ -924,6 +926,209 @@ public function document_link_access_type_table() {
+				'fallback' => __( 'Check PHP disable_functions', 'woocommerce-pdf-invoices-packing-slips' ),
+			),
+			'WP Memory Limit' => array(
+				/* translators: <a> tags */
+				'required' => sprintf( __( 'Recommended: 128MB (more for plugin-heavy setups<br/>See: %1$sIncreasing the WordPress Memory Limit%2$s', 'woocommerce-pdf-invoices-packing-slips' ), '<a href="https://docs.woocommerce.com/document/increasing-the-wordpress-memory-limit/" target="_blank">', '</a>' ),
+				'value'    => sprintf( 'WordPress: %s, PHP: %s', WP_MEMORY_LIMIT, $php_mem_limit ),
+				'result'   => $memory_limit > 67108864,
+			),
+			'allow_url_fopen' => array (
+				'required' => __( 'Allow remote stylesheets and images', 'woocommerce-pdf-invoices-packing-slips' ),
+				'value'	   => null,
+				'result'   => (bool) ini_get( 'allow_url_fopen' ),
+				'fallback' => __( 'allow_url_fopen disabled', 'woocommerce-pdf-invoices-packing-slips' ),
+			),
+			'fileinfo' => array (
+				'required' => __( 'Necessary to verify the MIME type of local images.', 'woocommerce-pdf-invoices-packing-slips' ),
+				'value'	   => null,
+				'result'   => extension_loaded( 'fileinfo' ),
+				'fallback' => __( 'fileinfo disabled', 'woocommerce-pdf-invoices-packing-slips' ),
+			),
+			'base64_decode'	=> array (
+				'required' => __( 'To compress and decompress font and image data', 'woocommerce-pdf-invoices-packing-slips' ),
+				'value'	   => null,
+				'result'   => function_exists( 'base64_decode' ),
+				'fallback' => __( 'base64_decode disabled', 'woocommerce-pdf-invoices-packing-slips' ),
+			),
+		);
+
+		if ( $imagick ) {
+			$gmagick_imagick_position = array_search( 'GMagick or IMagick', array_keys( $server_configs ) ) + 1;
+			$image_magick_config      = array(
+				'ImageMagick' => array(
+					'required' => __( 'Required for IMagick', 'woocommerce-pdf-invoices-packing-slips' ),
+					'value'    => ( $imagick && class_exists( '\\Imagick' ) ) ? esc_attr( \Imagick::getVersion()['versionString'] ) : null,
+					'result'   => $imagick,
+					'fallback' => __( 'ImageMagick library, integrated via the IMagick PHP extension for advanced image processing capabilities', 'woocommerce-pdf-invoices-packing-slips' ),
+				),
+			);
+
+			$server_configs = array_slice( $server_configs, 0, $gmagick_imagick_position, true ) + $image_magick_config + array_slice( $server_configs, $gmagick_imagick_position, null, true );
+		}
+
+		if ( $xc || $apc || $zop || $op ) {
+			$server_configs['opcache']['result'] = true;
+			if ( $xc ) {
+				$server_configs['opcache']['value'] = 'XCache ' . phpversion( 'xcache' );
+			} elseif ( $apc ) {
+				$server_configs['opcache']['value'] = 'APC ' . phpversion( 'apc' );
+			} elseif ( $zop ) {
+				$server_configs['opcache']['value'] = 'Zend OPCache ' . phpversion( 'Zend OPcache' );
+			} else {
+				$server_configs['opcache']['value'] = 'PHP OPCache ' . phpversion( 'opcache' );
+			}
+		}
+
+		if ( ! $server_configs['PHP version']['result'] ) {
```

Comment:
I believe this section can be removed, as promoting PHP 5 is no longer necessary or recommended.